content/zh/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -480,7 +480,7 @@ web-2   k8s.gcr.io/nginx-slim:0.8
 `web-0` has had its image updated, but `web-0` and `web-1` still have the original 
 image. Complete the update by deleting the remaining Pods.
 
-​```shell
+```shell
```

Comment:
I'd expect that we didn't change the _zh_ locale documents in this PR. Have I got that right?